pom.xml

```diff
@@ -2146,7 +2145,7 @@
         <identity.extension.utils>1.0.8</identity.extension.utils>
 
         <!-- Hash Provider Versions-->
-        <hashprovider.pbkdf2.version>0.1.1</hashprovider.pbkdf2.version>
+        <hashprovider.pbkdf2.version>0.1.2-SNAPSHOT</hashprovider.pbkdf2.version>
```

Comment:
need to update the version here with the released version after merging https://github.com/wso2-extensions/identity-hash-provider-pbkdf2/pull/10